site/tutorials/keptn-multistage-qualitygates-09.md

```diff
@@ -162,12 +169,12 @@ We are now ready to kick off a new deployment of our test application with Keptn
 
     <!-- command -->
     ```
-    keptn trigger delivery --project="pod-tato-head" --service=helloservice --image="gabrieltanner/hello-server" --tag=v0.1.1
+    keptn trigger delivery --project="podtatohead" --service=helloservice --image="docker.io/jetzlstorfer/helloserver" --tag=0.1.1
```

Comment:
@johannes-b Not sure if it's a good idea that we are hosting an image used in our tutorials on a "private" account on DockerHub that we can not access. Do we have any alternatives available?

Reply:
We've decided to move those images to a Keptn-related organization. We will update this PR with a comment soon.

Reply:
Let me know when the new Docker image is available then I will change the image all over the tutorial.

Reply:
@christian-kreuzberger-dtx Any progress here?

Reply:
This is planned for one of the next sprints, but no progress yet.

Reply:
We have finished the changes on the quickstart, please use the following images and tags:

* normal version: `ghcr.io/podtato-head/podtatoserver` - `v0.1.1` (don't forget to use the prefix `v`)
* slow version: `ghcr.io/podtato-head/podtatoserver` - `v0.1.2` (don't forget to use the prefix `v`)

Reply:
It should be resolved now!